semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/TagTraversal.scala

```diff
@@ -1,79 +1,39 @@
 package io.shiftleft.semanticcpg.language
 
+import io.shiftleft.codepropertygraph.generated.EdgeTypes
 import io.shiftleft.codepropertygraph.generated.nodes._
-import io.shiftleft.codepropertygraph.generated.{EdgeTypes, NodeTypes}
 import overflowdb.traversal._
 
 class TagTraversal(val traversal: Traversal[Tag]) extends AnyVal {
 
   def member: Traversal[Member] =
-    traversal
-      .in(EdgeTypes.TAGGED_BY)
-      .hasLabel(NodeTypes.MEMBER)
-      .sortBy(_.id)
-      .cast[Member]
+    traversal.in(EdgeTypes.TAGGED_BY).collectAll[Member]
```

Comment:
Not sorting anymore changes the order which may have influence on findings.

Reply:
I was going to add the sort at the usage side...

context (copied from the pr description):  I dropped the .sort because it's a bad default: it only works if the entire traversal is in memory.

Reply:
Ok but please make sure not to miss any usage sites.

Reply:
I changed my mind :) It's used in too many places in codescience...